rails-snow-queen/app/controllers/quotes_controller.rb

```diff
@@ -6,7 +6,7 @@ def new
   def create
     @quote = Quote.new quote_params
     if @quote.save
-      #mandar mailer 
+      QuoteMailer.new_quote(@quote).deliver
```

Comment:
Eventually, in production, you'll probably want to move this to a background worker (add to To-Do list), but in the meantime, doesn't this need to be `deliver_now`?

Reply:
added to to-do list